### PR TITLE
Feature/DTGB-415: Added styling for node preview form in Drupal.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [x] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,7 +58,7 @@ All Notable changes to `digipolisgent/drupal_theme_gent-base`.
 
 * DTGB-123: Implemented accessible hamburger-menu.
 
-## gent_base-8.x-1.1-alpha4 
+## gent_base-8.x-1.1-alpha4
```
Comment (Jeroen005) [inline CHANGELOG.md:61]: blank line

File: source/sass/11-base/contextual-links/_contextual-links.scss [hunk 2]
```diff
@@ -5,8 +5,12 @@
   .trigger,
   .trigger:focus,
   .trigger:hover {
-    background: $color-white url("../../../../contrib/gent_base/source/img/pencil.svg") no-repeat center center !important;
-    background-size: .75rem !important;
+    @include theme('background-color', 'color-secondary', 'contextual-link-background');
+
+    background-image: url("../../../../contrib/gent_base/source/img/pencil.svg");
+    background-repeat:no-repeat;
```
Comment (Jeroen005) [inline source/sass/11-base/contextual-links/_contextual-links.scss]: space

File: styleguide/components/11-base/fonts/_icons_template.template [hunk 3]
```diff
@@ -11,12 +11,12 @@ $id: random(1000);
 
 @font-face {
   font-family: "<%= fontName %>";
-  src: url('<%= fontPath %><%= fontName %>.eot?v='#{$id});
-  src: url('<%= fontPath %><%= fontName %>.eot?#iefix') format('eot'),
-  url('<%= fontPath %><%= fontName %>.woff2') format('woff2'),
-  url('<%= fontPath %><%= fontName %>.woff') format('woff'),
-  url('<%= fontPath %><%= fontName %>.ttf') format('truetype'),
-  url('<%= fontPath %><%= fontName %>.svg#<%= fontName %>') format('svg');
+  src: url('#{$styleguide-dir}/fonts/<%= fontName %>.eot?v='#{$id});
```
Comment (Jeroen005) [inline styleguide/components/11-base/fonts/_icons_template.template:14]: moet de id ook niet bij andere formats?

File: styleguide/components/11-base/fonts/_icons_template.template [hunk 4]
```diff
@@ -31,7 +31,7 @@ $id: random(1000);
   text-transform: none;
 }
 
-[class*='<%= cssClass%>'],
+[class^='<%= cssClass%>'],
```
Comment (Jeroen005) [inline styleguide/components/11-base/fonts/_icons_template.template]: Hierdoor werkt het enkel als de classs eerst staat, beter een * gebruiken

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (meirege) [reply CHANGELOG.md:61]: Fixed.

File: source/sass/11-base/contextual-links/_contextual-links.scss (hunk 2 above)
Comment (meirege) [reply source/sass/11-base/contextual-links/_contextual-links.scss]: Fixed.

File: styleguide/components/11-base/fonts/_icons_template.template (hunk 3 above)
Comment (meirege) [reply styleguide/components/11-base/fonts/_icons_template.template:14]: Agreed, added them.

File: styleguide/components/11-base/fonts/_icons_template.template (hunk 4 above)
Comment (meirege) [reply styleguide/components/11-base/fonts/_icons_template.template]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**